docs/glossary.md

```diff
@@ -87,6 +87,12 @@ Outputs from coinbase and stakebase transactions cannot be spent until the coinb
 
 Every block which is mined contains a single coinbase transaction. This transaction will only have one input, and that is newly created Decred which constitutes the PoW and Treasury portions of the block reward. Coinbase transactions are part of the regular transaction tree, which means that they will be rejected if proof-of-stake voters vote to reject the block which contains them. Decred created in coinbase transactions cannot be spent until the coinbase maturity period has passed.
 
+#### Coin type
+
+A constant, set for each cryptocoin for the BIP32 path. BIP-0032 introduces [HD Wallets](#hd-wallet). In the BIP32 path, every coin has an index. The index of Decred is 42. Registered coins are stored in a Satoshi Labs Improvement Proposal, in [SLIP-0044](https://github.com/satoshilabs/slips/blob/master/slip-0044.md). 
```

Comment:
>are stored in a Satoshi Labs Improvement Proposal, in [SLIP-0044](https://github.com/satoshilabs/slips/blob/master/slip-0044.md).

This wording is redundant. Suggest something like "..are stored in [SLIP-0044](https://github.com/satoshilabs/slips/blob/master/slip-0044.md)."

It would be nice to spell out the Satoshi Labs Improvement Proposal (SLIP) acronym here, but can't think of a non awkward way to do it. I think it's OK not to in this case. If the reader clicks on the link they'll see it spelled out. Alternately, we can use the current wording, but remove the 'in'. So, "in a Satoshi Labs Improvement Proposal, [SLIP-0044](https://github.com/satoshilabs/slips/blob/master/slip-0044.md)."